src/Makefile.am

```diff
@@ -43,6 +43,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/include        \
 
 bin_PROGRAMS = curl
 
+SUBDIRS = ../docs
```

Comment:
Was this the answer to the `SUBDIRS = ../docs` comment? Or to the other one, which you fixed?